priiloader/hacks_hash.ini

```diff
@@ -548,3 +548,15 @@ minversion=1
 amount=1
 hash=0x2c030000,0x418200ac,0x38000001,0x981f0010
 patch=0x480000b0
+[No-Block RAAE,408x,410x]
+maxversion=513
+minversion=417
+amount=1
+hash=0x28009ea3,0x4082009c
+patch=0x28009ea3,0x4082009c,0x4280006c
+[Remove IOS16 Disc Error]
+maxversion=4610
+minversion=417
+amount=1
+hash=0x68000001,0x68630010
+patch=0x68000002,0x6863424b
```

Comment:
what?!
whats wrong with what i suggested above? why change both the check of the upper & lower title id?
why xor with 0x3424? that only works because the upper title id check is changed, without it it'll break everything...

Reply:
It's not an XOR with 0x3424? It's 0x424b (BK). I chose this ID because any ID could be set as the requested IOS without error if that title is installed (whether it's a valid IOS number is irrelevant; someone tested this a while back), but I think we can confidently say that no titles use 00000002, since if that existed, it would be for a different platform.

Reply:
ok, what i said was a bit wrong, but my statement stays the same. its not a XOR with 3424 but XOR with 424B which in the end, changes nothing.

its still a XOR , and its XOR'ing with some random number.
the only reason it works is because you also changed the check on the upper title id as well

![image](https://user-images.githubusercontent.com/12701306/176303739-c0a779b6-7cf4-4076-83e4-e8a2fc835b12.png)
![image](https://user-images.githubusercontent.com/12701306/176304063-3b2fa38b-af14-4ab3-b939-cab059d926f4.png)

why the hell did you do this.
just change it to 68000001 & 68000002 so it compares it to system menu (which is most certainly invalid and should throw an error)